src/main/java/de/tum/in/www1/artemis/domain/ComplaintResponse.java

```diff
@@ -23,16 +25,42 @@
     @Size(max = 2000)
     private String responseText;
 
+    /**
+     * will be set as soon as the reviewer has submitted his complaint evaluation
+     */
     @Column(name = "submitted_time")
     private ZonedDateTime submittedTime;
 
+    /**
+     * will be set as soon as the evaluation of a complaint by a reviewer begins (possibly null for old complaint responses befor this column was added)
+     */
+    // ToDo add created time to liquibase (currently manually set via sql)
+    @Column(name = "created_time")
+    private ZonedDateTime createdTime;
+
+    @Transient
+    private boolean isCurrentlyLocked;
```

Comment:
How is this used? Maybe we should add a comment here.

Reply:
I see it is used in the client, but where and when is this calculated?

Reply:
The lock feature is not implemented yet.